### PR TITLE
search-engines: delist startpage

### PR DESCRIPTION
**Pull request by Mikaela**
Resolves: #1409

Preview: https://deploy-preview-1410--privacytools-io.netlify.com/providers/search-engines/#search

### CODE REVIEW

**Comment by Mikaela** [comment]
In case anyone finds here, but not the discussion, it's in two places https://github.com/privacytoolsIO/privacytools.io/issues/1409#issuecomment-542506807 and [our forum](https://forum.privacytools.io/t/candidate-for-delisting-discussion-startpage/284/6?u=mikaela) before the WIP (don't merge/PR) label gets removed and anything happens with this PR.

With luck I think we may even be able to close this issue and not delist Startpage, but it's too early to say.

File: _includes/sections/search-engines.html [hunk 1]
```diff
@@ -14,14 +14,6 @@ <h1 id="search" class="anchor"><a href="#search"><i class="fas fa-link anchor-ic
 github="https://github.com/asciimoo/searx"
 %}
 
-{% include cardv2.html
-title="StartPage - Netherlands"
-image="/assets/img/provider/StartPage.png"
```
Comment (djoate) [inline _includes/sections/search-engines.html]: If we end up delisting StartPage, we should also remove the asset (https://github.com/privacytoolsIO/privacytools.io/blob/master/assets/img/provider/StartPage.png)

Comment (Mikaela) [reply _includes/sections/search-engines.html]: Thanks, done

**Comment by Mikaela** [comment]
I also fixed the merge conflict and removed the WIP label, even while I am not sure if a concensus was reached, I am ~10 days behind on everything.

**Comment by Mikaela** [comment]
@privacytoolsIO/editorial Can we merge this? Or does @blacklight447-ptio want to suggest changes to add a note to explain the situation?

CC: @LizMcIntyre

**Comment by LizMcIntyre** [comment]
> @privacytoolsIO/editorial Can we merge this? Or does @blacklight447-ptio want to suggest changes to add a note to explain the situation?
> 
> CC: @LizMcIntyre

Hi @Mikaela  @blacklight447-ptio I believe *something* should be posted to note the removal. I'm thinking a short comment is best. Here's the latest I recommended in the discussion thread:

**The Startpage search engine is de-listed until it is fully transparent about its [new ownership](https://www.reddit.com/r/StartpageSearch/comments/dicp03/startpage_is_now_owned_by_an_advertising_company/) and data processing.**

People can click the link to get the details.

I believe that once PrivacytoolsIO recommends a service, it should indicate that a service is removed and why it is removed. This will help alert those who use a recommended service that has lost trust. How else might they know?

A press release and other media might also be a good idea because some people might not return to see a warning.

**Comment by Mikaela** [comment]
@privacytoolsIO/editorial & @LizMcIntyre, please rereview changes at https://github.com/privacytoolsIO/privacytools.io/pull/1410/files + consider can we merge it.

I have no comment on the rest at the moment.

File: _includes/sections/search-engines.html [hunk 2]
```diff
@@ -14,13 +14,7 @@ <h1 id="search" class="anchor"><a href="#search"><i class="fas fa-link anchor-ic
...
-description="StartPage is a search engine that provides Google search results with complete privacy protection. Behind StartPage is a European company that has been obsessive about privacy since 2006."
-website="https://www.startpage.com/"
-forum="https://forum.privacytools.io/t/discussion-startpage/284"
-%}
+<p><strong>The Startpage search engine is de-listed until it is fully transparent about <a href="https://www.reddit.com/r/StartpageSearch/comments/dicp03/startpage_is_now_owned_by_an_advertising_company/">its new ownership</a> and data processing.</strong></p>
```
Comment (djoate) [inline _includes/sections/search-engines.html]:
```suggestion
<p><strong>The Startpage search engine is delisted until it is fully transparent about <a href="https://www.reddit.com/r/StartpageSearch/comments/dicp03/startpage_is_now_owned_by_an_advertising_company/">its new ownership</a> and data processing.</strong></p>
```

Comment (nitrohorse) [reply _includes/sections/search-engines.html]: Yeah, I agree; the warning in the middle does look a bit odd.

**Comment by blacklight447-ptio** [comment]
Tbh, do we want the post delisting on the website now? It goes a bit against what we normally do, i would personally opt to just remove it, and put out a small blog post.

**Comment by nitrohorse** [comment]
You’re right, that would make more sense overall I think.

**Comment by Mikaela** [comment]
Trai comments:

> Mikaela, if we make an announcement out of it, let's also include the r/PTIO post that Liz referenced in your linked pull request: https://www.reddit.com/r/StartpageSearch/comments/dicp03/startpage_is_now_owned_by_an_advertising_company/
It's more informative, and includes links to other posts that include SP's response (such as it is).
It'll let readers go as deep as they'd like to go.

So should I:

* revert https://github.com/privacytoolsIO/privacytools.io/pull/1410/commits/ff5ef961f1fb4ce34948ea046d85358c1e3ad6fe ?
* move the warning to bottom and add link to Privacytools subreddit?

**Comment by blacklight447-ptio** [comment]
I would just remove startpage, add nothing else, then publish a blog explaining the removal.

**Comment by LizMcIntyre** [comment]
> I would just remove startpage, add nothing else, then publish a blog explaining the removal.

That could work, too. Less disruptive to the flow and tradition, as pointed out.

**Pull request merged**